src/Pages/SignIn.tsx

```diff
@@ -1,16 +1,92 @@
-import { Link } from "react-router-dom"
-import styled from "styled-components"
+import OnboardingPage from './OnboardingPage';
```

Comment:
Is the `OnboardingPage` confusing? It's a container component that receives `children` as a prop. It's for DRY code implementation because there is a similarity with SignIn and SignUp pages. Maybe the component name that creates the confusion? Or I don't understand your comment.